accounts-db/accounts-hash-cache-tool/src/main.rs

```diff
@@ -226,24 +226,17 @@ fn do_diff_files(file1: impl AsRef<Path>, file2: impl AsRef<Path>) -> Result<(),
             };
             entries.push(entry);
         }
+
+        // entries in the file are sorted by pubkey then slot,
```

Comment:
Do we want the entries to be sorted by keys? 
HashMap may not preserve the key ordering.  Maybe BTreeMap?

Reply:
Never mind. I see that HashMap is fine for the later diff.